waiter/src/waiter/core.clj

```diff
@@ -98,6 +98,7 @@
                      "app-name" :app-name-handler-fn
                      "apps" {"" :service-list-handler-fn
                              ["/" "instances"] :instances-list-handler-fn
+			     ["/" :instance-id "/signal"] :signal-handler-fn
```

Comment:
How do you feel about `/apps/<service-id>/signal` and sending the instance id, signal and timeout value in a json payload for a POST request?